kmcos/run/__init__.py

```diff
@@ -1389,36 +1405,53 @@ def get_species_coords(self, filename_csv="", export_csv=True):
         'species' in the function is a dictionary that contains the names of the species
             EX: {'CO': 'carbon', 'empty': ''}
 
-        """
-        if filename_csv == "" and export_csv==True:
-            filename_csv = "species_coords.csv"
+        'matrix_format' has two types of options: meshgrid and cartesian. Cartesian return as a csv with (x,y,z) format, and the meshgrid format
```

Comment:
Please check this line. This looks like it is supposed to be in the comments, but it looks like it is not in the comments.